tests/genai/judges/test_judges.py

```diff
@@ -33,13 +35,13 @@ def test_databricks_judges_are_importable():
     assert judges.custom_prompt_judge == custom_prompt_judge
 
 
-def create_test_feedback(value: str) -> Feedback:
+def create_test_feedback(value: str, error: Optional[str] = None) -> Feedback:
     return Feedback(
         name="test_feedback",
         source=AssessmentSource(source_type=AssessmentSourceType.LLM_JUDGE, source_id="databricks"),
         rationale="Test rationale",
         metadata={},
-        value=FeedbackValue(value=value, error=None),
+        value=FeedbackValue(value=value, error=error),
         valid=True,
```

Comment:
```suggestion
        value=value,
        error=error,
```

This should be the correct constructor (and I guess this is why the test pass falsely)